sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -148,18 +175,13 @@ def _populate_metric_results(self, response):
       metrics_by_name[metric_key][tentative_or_committed] = metric
 
     # Now we create the MetricResult elements.
-    result = []
     for metric_key, metric in iteritems(metrics_by_name):
       attempted = self._get_metric_value(metric['tentative'])
       committed = self._get_metric_value(metric['committed'])
-      if attempted is None or committed is None:
-        continue
       result.append(MetricResult(metric_key,
```

Comment:
Perhaps the MetricResult, or the MetricKey should somehow indicate that the origin is the system. I see that we have `namespace` set to dataflow/v1b3. I wonder if that's enough...

Reply:
no need for that. Since the URNs do this.